CHANGELOG.md

```diff
@@ -1,11 +1,21 @@
 # Changelog
+
 All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+## [0.7.0]
+
+- Allow transaction with undefined 'to' fields, in signTransaction ([#98](https://github.com/MetaMask/eth-ledger-bridge-keyring/pull/98))
```

Comment:
The bug fixed by this commit was introduced in the previous commit - it didn't exist in the previous release. We can omit this. Or maybe list the link for this PR alongside the "Update eth-ledger-bridge-keyring to support EIP-1559 transactions" PRs.